x/wasm/keeper/query_plugins.go

```diff
@@ -84,13 +84,16 @@ func (q QueryHandler) GasConsumed() uint64 {
 
 type CustomQuerier func(ctx sdk.Context, request json.RawMessage) ([]byte, error)
 
+type stargateQuerierFn func(ctx sdk.Context, request *wasmvmtypes.StargateQuery) ([]byte, error)
+type grpcQuerierFn func(ctx sdk.Context, request *wasmvmtypes.GrpcQuery) (proto.Message, error)
```

Comment:
Is it safe that those are private? Can users still set the queries by just implicitly fulfilling the interface?

Reply:
Yes, I think that's fine.